### ISSUE TITLE
All these new is available only on?

### ISSUE THREAD

**Issue by RAJU2529** [MicrosoftDocs/windows-itpro-docs#3814]
What will happen, If the users upgrades from existing Windows 10 to 1903 , after upgraded, whether users will get all the new features as mentioned in this article.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 095d9daa-2d4e-7442-13a1-79d15b313ad7
* Version Independent ID: c33e2393-9f53-d896-e154-42058ecf4dbf
* Content: [What's new in Windows 10, version 1903](https://docs.microsoft.com/en-us/windows/whats-new/whats-new-windows-10-version-1903)
* Content Source: [windows/whats-new/whats-new-windows-10-version-1903.md](https://github.com/MicrosoftDocs/windows-itpro-docs/blob/master/windows/whats-new/whats-new-windows-10-version-1903.md)
* Product: **w10**
* Technology: **windows**
* GitHub Login: @greg-lindsay
* Microsoft Alias: **trudyha**

**Comment by e0i** [comment]
@officedocsbot assign @e0i

**Comment by e0i** [comment]
@RAJU2529 

The issues section of this repository is intended for documentation and content related issues.

Please open a support ticket via the link below so that that your question gets answered quickly.

[Windows 10 Support](https://support.microsoft.com/en-us/hub/4338813/windows-help?os=windows-10)

Thank you.

**Comment by e0i** [comment]
@officedocsbot close

**Comment by illfated** [comment]
@RAJU2529 : Your question is too general to be answered, even by support persons. The only way to find out what the effect is of updating Windows 10 to Version 1903 from earlier versions is to test it yourself on one or a couple of computers with no importance to production value (that is, test computers with no important data or production applications). Furthermore, your question is something every IT professional is asked by their leaders before upgrades take place. Almost every company with important production configurations on their Windows 10 computers need local support from their local IT personnel to make sure that their computers will be just as stable as before, when updates and upgrades are applied. Such changes will, more often than not, be specific to the local configuration in all of the computers. Also, it will depend heavily on whether or not there are any custom applications or custom configurations not available to Microsoft.

Myself, I have updated to 1903 on my personal laptop computer (Windows 10 Home) and my workplace computer (Windows 10 Pro), but I do not have much specific software packages installed that I cannot upgrade if Version 1903 brings unexpected changes that I did not read about before updating. Only you yourself will be able to test, verify and estimate if the update will be smooth and without consequences in your production environment (company computers). You simply need to test in your own environment to see if the changes will have any positive, negative or zero effects on the computers used in your local production environment. Just make sure you have access to test equipment to check what happens before rolling out the update.
